pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks.yaml

```diff
@@ -82,12 +82,17 @@ spec:
         #!/usr/bin/env bash
         set -e
 
+        func_file="$(workspaces.source.path)/func.yaml"
+        if [ "$(params.SOURCE_SUBPATH)" != "" ]; then
+          func_file="$(workspaces.source.path)/$(params.SOURCE_SUBPATH)/func.yaml"
+        fi
+
```

Comment:
Could you please move these lines right before, so the changes are on a fewer places
```
echo "--> Saving 'func.yaml'"
```

and could you please surrund this in:
```
 ############################################
        ##### Added part for Knative Functions #####
############################################

...new stuff here

############################################

```